includes/class-wc-siftscience-html.php

```diff
@@ -27,6 +27,119 @@ class WC_SiftScience_Html {
 		public const WC_CHECKBOX_ELEMENT   = 'checkbox';
 		public const WC_SECTIONEND_ELEMENT = 'sectionend';
 		public const WC_CUSTOM_ELEMENT     = 'custom';
+		
+		public function __call( $name, $args ) {
+			if ( 'create_element' === $name ) {
+
+				switch ( count( $args ) ) {
```

Comment:
This looks like it's equivalent exactly to this:

```
public function create_element( $type, $id, $label = '', $desc = '', $element_options = array() )
```

Reply:
yep this is called overloading much more efficient than default parameters